ChainSafe/gossamer
# feat(cmd/gossamer): implement --telemetry-url parameter

##  Changes

- Added `--telementry-url` parameter

## Tests


- Use https://github.com/paritytech/substrate-telemetry to setup telemetry server. Run telementry backend (`telemetry_core` and `telemetry_shard`) and frontend (`yarn start`).
- Run gossamer with telemetry url parameter.  This parameter is a StringSlice and can be passed multiple times.
- `./bin/gossamer --chain polkadot --name 'Kishans Test Node' --telemetry-url 'ws://localhost:8001/submit 0' --telemetry-url 'wss://telemetry.polkadot.io/submit/ 0'`
- Your node should show up in your local telemetry server after waiting a bit.
```

```

## Issues



- Closes #1502

## Primary Reviewer 



- @edwardmack
